src/i18n/en.json

```diff
@@ -1454,7 +1454,18 @@
       "investment-submitted-msg": "Invested successfully. Your account will be funded with {{asset}} once the sale succeeds",
       "offer-canceled-msg": "Your offer was successfully cancelled",
       "requires-verification-title": "Verification required",
-      "requires-verification-desc": "Please, verify your account to invest in this sale"
+      "requires-verification-desc": "Please, verify your account to invest in this sale",
+      "deposit-btn": "Deposit",
+      "fees": "Fees",
+      "fixed-fee": "fixed fee",
+      "percent-fee": "percent fee",
+      "total": "Total",
+      "total-fee": "Total fee",
+      "total-amount": "Total amount",
+      "continue-btn": "Continue",
+      "recheck-form": "Please recheck the form before proceeding",
+      "transaction-fees": "Transaction fees",
+      "no-transaction-fees": "No fees for this transaction"
```

Comment:
introduce suffixes, so the reader can understand roles of the translations. for example
```
transaction-fees-heading
no-transaction-fee-msg
total-fee-label
total-label
total-amount-label
```

Reply:
edited